yjit/src/asm/mod.rs

```diff
@@ -216,6 +213,8 @@ impl CodeBlock {
         self.page_end_reserve = old_page_end_reserve;
     }
 
+    #[cfg(target_arch = "aarch64")]
+    #[cfg(not(test))]
```

Comment:
Hmm I think this might apply to x86 as well, wouldn't really want to guard this bit like that

Reply:
It isn't used by x86. That's why I put the guard.